src/app/core/services/sync-data.service.js

```diff
@@ -87,9 +87,7 @@ export default function (app) {
       }
 
       this.changeUserRole = (uid, role) => {
-        console.log(role);
         $rootScope.listOfUsers[uid].role = role;
```

Comment:
We're not able to reach the element in the `listOfUsers` by the key `uid` now, because we decided to transform listOfUsers as Object with key `uid` into Array without any keys.
Probably, we should leave original listOfUsers as an Object (as we get it from Firebase) but create a separate array for the table?

Reply:
Yes, it would be better to create separate array especially for admin page, because we don't need all parameters which listOfUsers contains. 
At this branch I used listOfUsers as an object because I don't have array yet. As soon as I get data in array format I'll rewrite particular functions.